src/rgw/motr/gc/gc.h

```diff
@@ -57,6 +58,90 @@ class MotrGC : public DoutPrefixProvider {
   };
   std::vector<std::unique_ptr<MotrGC::GCWorker>> workers;
 
+  struct Meta {
```

Comment:
Hi sorry I noticed it just now, can we keep `Meta` and `motr_gc_obj_info` out of MotrGC class?

Reply:
Moved them outside of the class